src/hotspot/share/utilities/macros.hpp

```diff
@@ -358,6 +358,21 @@
 #define NOT_CHECK_UNHANDLED_OOPS(code)  code
 #endif // CHECK_UNHANDLED_OOPS
 
+// Enable collection of TaskQueue statistics.
+// Enabled by default in debug builds.  Otherwise, disabled by default.
+#ifndef TASKQUEUE_STATS
+#ifdef ASSERT
+#define TASKQUEUE_STATS 1
+#else
+#define TASKQUEUE_STATS 0
+#endif // ASSERT
+#endif // TASKQUEUE_STATS
+#if TASKQUEUE_STATS
+#define TASKQUEUE_STATS_ONLY(code) code
+#else
+#define TASKQUEUE_STATS_ONLY(code)
+#endif // TASKQUEUE_STATS
+
```

Comment:
Duplicated definition in `TaskQueue.hpp`
https://github.com/openjdk/jdk/blob/ab1dbd4089a1a15bdf1b6b39994d5b1faacc40ab/src/hotspot/share/gc/shared/taskqueue.hpp#L39-51 should be removed.

Reply:
Oops.  Fixed.